addons/mail/static/src/components/chatter_container/chatter_container.js

```diff
@@ -99,6 +99,7 @@ class ChatterContainer extends Component {
 
 Object.assign(ChatterContainer, {
     components,
+    defaultProps: {},
```

Comment:
Or enterprise could do:
```js
Object.assign(chatterContainer.defaultProps || {}, {
   // ...
})
```

_(see added `|| {}`)_

Note that if we could use `...` on object, syntax would be nice regardless of evaluation being `...{}` or `...undefined`.